deploy/sre-prometheus/100-kube-controller-manager-crashlooping.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-kube-controller-manager-crashlooping
+    role: alert-rules
+  name: sre-managed-kube-controller-manager-crashlooping
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-kube-controller-manager-crashlooping
+    rules:
+    - alert: KubeControllerManagerCrashlooping
+      # This is a stop-gap alert until OCPBUGS-10761 is investigated and fixed
+      # There has been cases where the KCM operator does not enter Down state and results in a few other issues
+      # This is an attempt to catch that before it causes more problems
+      # https://issues.redhat.com/browse/OCPBUGS-10761
+      expr: |
+        sum(kube_pod_container_status_waiting_reason{reason="CrashLoopBackOff", namespace="openshift-kube-controller-manager",pod=~"kube-controller-manager-.*",job="kube-state-metrics"}) >= 2
+      for: 60m
+      labels:
+        severity: critical
```

Comment:
Whilst we are experimenting with this alert and its maturity is immature (per the line below) do we want to set this to warning initially and observe its noisiness in Telemetry first? It also then doesn't necessarily require a SOP written before going out.

Reply:
I wasn't sure about the maturity and immature seemed like a good idea so that is an excellent plan.
I will drop the severity to warning.

Reply:
Fixed in latest changes.